### PR TITLE
Fix typos in `meetings`

### PR DESCRIPTION
**Pull request by stakx**
Submitting this as a separate PR since I wasn't sure whether you're interested at all in fixing "archived" documents.

### CODE REVIEW

File: meetings/2015/LDM-2015-01-21.md [hunk 1]
```diff
@@ -195,7 +195,7 @@ Metaprogramming
...
 
-Extention methods and partial classes both feel like features that could grow into allowing *generated* parts of source code to merge smoothly with *hand-written* parts. But if generated parts are themselves the result of language syntax - e.g. attributes in source code, then things quickly get messy from a tooling perspective. A keystroke in file A may cause different code to be generated into file B by some custom program, which in turn may change the meaning of A. Not a feedback loop we're eager to have to handle in real time at 20 ms keystroke speed!
+Extension methods and partial classes both feel like features that could grow into allowing *generated* parts of source code to merge smoothly with *hand-written* parts. But if generated parts are themselves the result of language syntax - e.g. attributes in source code, then things quickly get messy from a tooling perspective. A keystroke in file A may cause different code to be generated into file B by some custom program, which in turn may change the meaning of A. Not a feedback loop we're eager to have to handle in real time at 20 ms keystroke speed!
```
Comment (stakx) [inline meetings/2015/LDM-2015-01-21.md]: Only change here: `Extention` &rarr; `Extension` (first word)

File: meetings/2017/LDM-2017-01-11.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ When there are expression variables in the case header, in the case body forbid:
 Drop the subbullets for now, and maybe we can relax later
 
 
-Out of all these approaches, we still think expanding the lifeltime (but not the scope) has the lowest risk. Fallout work is likely in the debugger, which will have a suboptimal experience in these scenarios. THis work is probably puntable.
+Out of all these approaches, we still think expanding the lifetime (but not the scope) has the lowest risk. Fallout work is likely in the debugger, which will have a suboptimal experience in these scenarios. This work is probably puntable.
```
Comment (stakx) [inline meetings/2017/LDM-2017-01-11.md]: No idea what "puntable" means, so I left it as is.

File: meetings/2015/LDM-2015-01-21.md (hunk 1 above)
Comment (jcouv) [reply meetings/2015/LDM-2015-01-21.md]: Thanks
It's strange that the github diff doesn't highlight that word...

File: meetings/2016/LDM-2016-07-13.md [hunk 3]
```diff
@@ -84,7 +84,7 @@ We think no. the existence of a `Deconstruct` method should not imply conversion
 
 We could consider letting user defined implicit conversion imply `Deconstruct`. It leads to some convenience, but makes for a less clean correspondence with consumption code.
 
-Let's keep it separate. If you want a type to be both deconstructible and convertible to tuple, you need to specify both.
```
Comment (jcouv) [inline meetings/2016/LDM-2016-07-13.md]: Nit: I think either is ok.

**Pull request merged**